Jellyfin.Drawing.Skia/SkiaEncoder.cs

```diff
@@ -358,15 +388,15 @@ private SKBitmap GetBitmap(string path, bool cropWhitespace, bool autoOrient, Im
 
         private SKBitmap OrientImage(SKBitmap bitmap, SKEncodedOrigin origin)
         {
-            //var transformations = {
+            // var transformations = {
```

Comment:
I think it's fine to remove this commented out code.

Reply:
Addressed in  4c30557